aiida_mlip/calculations/base.py

```diff
@@ -217,7 +217,7 @@ def prepare_for_submission(
             cmd_line["arch"] = architecture
 
         model_path = None
-        if "model" in self.inputs:
+        if "model" in self.inputs and self.inputs.model is not None:
```

Comment:
> At the moment it looks for model in the config, and if it doesn't find it, it will download a model based on the `"arch"`, but what if `"arch" is None` too?

yeah good point, if it's None it will use the januse default at the moment.
I consider this part of the defaults issue I guess